inspect_activations/Varying_optimizers/varying_optimizers_results.py

```diff
@@ -0,0 +1,66 @@
+| 18 | tanh_relu_tanh	|  97.21	|  10.32	|  97.88	|  97.88	| 
+| 19 | tanh_relu_sigmoid	|  94.53	|  10.09	|  97.78	|  97.78	| 
+| 20 | tanh_relu_selu	|  97.96	|  91.49	|  98.15	|  98.15	| 
+| 21 | tanh_tanh_relu	|  97.94	|  9.8	|  67.63	|  67.63	| 
+| 22 | tanh_tanh_tanh	|  97.09	|  11.35	|  98.24	|  98.24	| 
+| 23 | tanh_tanh_sigmoid	|  94.36	|  9.8	|  97.98	|  97.98	| 
+| 24 | tanh_tanh_selu	|  97.90	|  9.8	|  98.14	|  98.14	| 
+| 25 | tanh_sigmoid_relu	|  37.05	|  9.8	|  9.8	|  9.8	| 
+| 26 | tanh_sigmoid_tanh	|  95.02	|  11.35	|  98.11	|  98.11	| 
+| 27 | tanh_sigmoid_sigmoid	|  84.39	|  75.27	|  98.01	|  98.01	| 
+| 28 | tanh_sigmoid_selu	|  97.04	|  88.55	|  98.17	|  98.17	| 
+| 29 | tanh_selu_relu	|  97.95	|  9.8	|  9.8	|  9.8	| 
+| 30 | tanh_selu_tanh	|  97.38	|  10.32	|  97.9	|  97.9	| 
+| 31 | tanh_selu_sigmoid	|  94.76	|  9.8	|  97.77	|  97.77	| 
+| 32 | tanh_selu_selu	|  97.92	|  9.8	|  98.18	|  98.18	| 
+| 33 | sigmoid_relu_relu	|  75.94	|  9.8	|  19.92	|  19.92	| 
+| 34 | sigmoid_relu_tanh	|  94.09	|  9.8	|  97.98	|  97.98	| 
+| 35 | sigmoid_relu_sigmoid	|  85.04	|  9.8	|  97.4	|  97.4	| 
+| 36 | sigmoid_relu_selu	|  96.16	|  94.58	|  98.24	|  98.24	| 
+| 37 | sigmoid_tanh_relu	|  96.19	|  9.8	|  78.18	|  78.18	| 
+| 38 | sigmoid_tanh_tanh	|  93.95	|  9.8	|  98.1	|  98.1	| 
+| 39 | sigmoid_tanh_sigmoid	|  84.94	|  9.8	|  97.8	|  97.8	| 
+| 40 | sigmoid_tanh_selu	|  96.22	|  9.8	|  98.38	|  98.38	| 
+| 41 | sigmoid_sigmoid_relu	|  9.80	|  9.8	|  9.8	|  9.8	| 
+| 42 | sigmoid_sigmoid_tanh	|  91.80	|  10.1	|  97.77	|  97.77	| 
+| 43 | sigmoid_sigmoid_sigmoid	|  79.16	|  20.79	|  97.46	|  97.46	| 
+| 44 | sigmoid_sigmoid_selu	|  94.35	|  58.23	|  98.13	|  98.13	| 
+| 45 | sigmoid_selu_relu	|  86.61	|  9.8	|  70.42	|  70.42	| 
+| 46 | sigmoid_selu_tanh	|  94.43	|  9.8	|  98.09	|  98.09	| 
+| 47 | sigmoid_selu_sigmoid	|  89.79	|  9.8	|  97.81	|  97.81	| 
+| 48 | sigmoid_selu_selu	|  96.60	|  9.8	|  98.41	|  98.41	| 
+| 49 | selu_relu_relu	|  86.57	|  9.8	|  9.8	|  9.8	| 
+| 50 | selu_relu_tanh	|  97.11	|  9.58	|  98.07	|  98.07	| 
+| 51 | selu_relu_sigmoid	|  94.31	|  8.92	|  97.88	|  97.88	| 
+| 52 | selu_relu_selu	|  97.80	|  91.14	|  98.32	|  98.32	| 
+| 53 | selu_tanh_relu	|  97.74	|  9.8	|  67.53	|  67.53	| 
+| 54 | selu_tanh_tanh	|  96.73	|  10.32	|  98.09	|  97.9	| 
+| 55 | selu_tanh_sigmoid	|  94.12	|  9.8	|  97.9	|  98.09	| 
+| 56 | selu_tanh_selu	|  97.77	|  27.33	|  98.22	|  98.22	| 
+| 57 | selu_sigmoid_relu	|  85.24	|  19.22	|  49.39	|  98.01	| 
+| 58 | selu_sigmoid_tanh	|  94.52	|  36.18	|  98.15	|  98.04	| 
+| 59 | selu_sigmoid_sigmoid	|  91.46	|  80.74	|  98.04	|  98.15	| 
+| 60 | selu_sigmoid_selu	|  96.68	|  81.01	|  98.3	|  98.3	| 
+| 61 | selu_selu_relu	|  97.84	|  9.8	|  50.23	|  50.23	| 
+| 62 | selu_selu_tanh	|  96.77	|  11.35	|  88.35	|  88.35	| 
+| 63 | selu_selu_sigmoid	|  94.27	|  9.8	|  97.85	|  97.85	| 
+| 64 | selu_selu_selu	|  97.78	|  9.8	|  98.14	|  98.14	|
```

Comment:
@rajasekharmekala The RMSProp is underperforming so many times
But cant say for sure because there is only one experiment per setting.
Let us not focus on this for some time

Reply:
ok